packages/codewarsClassifier/marks.ts

```diff
@@ -0,0 +1,49 @@
+export const marksMapping = {
+    '8 kyu': 1,
+    '7 kyu': 1,
+    '6 kyu': 4,
+    '5 kyu': 5,
+    '4 kyu': 10,
+    '3 kyu': 15,
+    '1 kyu': 20
+}
+
+export const targetKatas = [
+  '56f935002e6c0d55fa000d92',
```

Comment:
Per our discussion, we decided to keep names for reference. Why did you decide to remove them?

Reply:
because we don't use it, but names are present in unit tests mock